contents/blog/best-mobile-app-ab-testing-tools.mdx

```diff
@@ -0,0 +1,303 @@
+---
+date: 2023-11-21
+title: The 9 best mobile app A/B testing tools
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/open-source-testing-tools/testinghog.png
+featuredImageType: full
+author:
+  - lior-neu-ner
+category: General
+tags:
+  - Guides
+  - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+Also known as split testing or [multivariate testing](/product-engineers/what-is-multivariate-testing-examples), A/B testing enables you to measure the impact of product changes on your mobile apps.
+
+There are many A/B testing tools out there, but each one is built for a specific target audience in mind. In this guide, we’ll explore these options so that you can decide which is best for you.
+
+## 1. PostHog
+
+![PostHog - best ab testing tools](../images/blog/best-mobile-app-ab-testing-tools/posthog.png)
+
+- **Best for:** Engineering and product teams
+- **Free tier:** <span className="text-green text-lg">✔</span>
+- **Transparent pricing:** <span className="text-green text-lg">✔</span>
+- **Open source:** <span className="text-green text-lg">✔</span>
+- **Get started without talking to sales:** <span className="text-green text-lg">✔</span>
```

Comment:
These are good, but could we include some A/B testing features here too? Potential options:

- Stats engines (Bayesian vs Frequentist)
- Local evaluation
- Multivariate tests
- JSON payloads
- Target size
- Custom goals
- Secondary metrics
- Statistical significance calculation
- Recommended run time
- Alerts

Too little to do with A/B testing at the moment

Reply:
I spent some time digging into our competitors websites and docs to figure out an angle for this. 

Most of the features in this list/we have are table stakes. All the A/B testing products have these.

I had a look also at Andy's blog [6 best free and open-source A/B testing tools](https://posthog.com/blog/best-open-source-ab-testing-tools) to see if I could do a similar "Features and Benefits" section like he did:

There is some minor differentiation (e.g. statsig's multi-arm bandit, taplytics no-code builder), but these are already called out in the intros. So I dont think theres a strong enough differentation between products to have this section. Even looking at Andy's blog, the feature and benefits section aren't really that clear e.g. things like `Lightweight SDKs for speed` or `Accurate user targeting` is pretty standard across products

I'm open to other ideas on this!